service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsController.java

```diff
@@ -143,7 +146,7 @@ private String buildProceedingLabel(HearingBooking hearingBooking) {
             .collect(Collectors.toList());
     }
 
-    private List<Document> generateAndUploadDocuments(Map<String, Object> templatePlaceholders,
+    private List<Document> generateAndUploadDocuments(DocmosisNoticeOfProceeding templatePlaceholders,
```

Comment:
could we move this into `NoticeOfProceedingsService`?

Reply:
agree, don't know why it was here!. Moved it now.

Reply:
as discussed, i am going to leave that change out of this PR.